### PR TITLE
TE-13.1: Add Arista support for CoreFileCheck functionality and fix an issue with configuring parent L3 interface

### PR DESCRIPTION
**Pull request by sambhavjain-204**
- Adding support for vendor Arista for the `coreFileCheck` functionality in this test.
- Fixing the issue with configuring L3 parent interface on the lines of other similar test TE-13.2

### CODE REVIEW

**Review by rszarecki — DISMISSED** [review]

**Review by mohanasm — CHANGES_REQUESTED** [review]

**Review by rszarecki — CHANGES_REQUESTED** [review]
Please have a look at line 300.
I belive it should be : "for i := 0; i <= nextHopCount; i++ {"
since change in line 284 "vlanID := uint16(i)" makes vlan starting form 0.

File: feature/experimental/gribi/ate_tests/route_addition_during_failover_test/route_addition_during_failover_test.go [hunk 1]
```diff
@@ -279,9 +281,12 @@ func generateSubIntfPair(t *testing.T, dut *ondatra.DUTDevice, dutPort *ondatra.
 	nextHops := []string{}
 	nextHopCount := 63 // nextHopCount specifies number of nextHop IPs needed.
 	for i := 0; i <= nextHopCount; i++ {
-		vlanID := uint16(i) + 1
+		vlanID := uint16(i)
```
[the remark this answers is not in the document]
Comment (sambhavjain-204) [reply feature/experimental/gribi/ate_tests/route_addition_during_failover_test/route_addition_during_failover_test.go]: I have reverted the changes for `vlanID` and `Index`

**Review by rszarecki — APPROVED** [review]

**Pull request merged**